src/Microsoft.Graph/Models/Generated/ContactFolder.cs

```diff
@@ -64,6 +50,20 @@ public partial class ContactFolder : Entity
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "multiValueExtendedProperties", Required = Newtonsoft.Json.Required.Default)]
         public IContactFolderMultiValueExtendedPropertiesCollectionPage MultiValueExtendedProperties { get; set; }
     
+        /// <summary>
```

Comment:
@MIchaelMainer Any idea why we are seeing these updates?  I don't see any changes.